config/settings.yml

```diff
@@ -1051,6 +1051,11 @@
       :starting_timeout: 20.minutes
       :poll: 10.seconds
       :memory_threshold: 0.megabytes
+    :amazon_agent_manager_worker:
+      :starting_timeout: 20.minutes
+      :heartbeat_timeout: 30.minutes
```

Comment:
Why is the heartbeat time out so high?

Reply:
Mentioned above. Give worker enough time to launch an agent.